main.py

```diff
@@ -1,10 +1,44 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Player.objects.all().delete()
+    Guild.objects.all().delete()
+
+    with open("players.json", "r") as players_file:
+        players_data = json.load(players_file)
+
+    for player_name, player_data in players_data.items():
+        race = Race.objects.get_or_create(
+            name=player_data["race"]["name"],
+            description=player_data["race"]["description"]
+        )[0]
```

Comment:
Perform unpacking here and in the other places instead of indexing for better readability
```suggestion
        race, _ = Race.objects.get_or_create(
            name=player_data["race"]["name"],
            description=player_data["race"]["description"]
        )
```